caf-audit-service-container/pom.xml

```diff
@@ -320,6 +324,15 @@
                                                 <unpack>true</unpack>
                                                 <useProjectArtifact>false</useProjectArtifact>
                                             </dependencySet>
+                                            <!-- Including the healthcheck web application. -->
+                                            <dependencySet>
+                                                <outputDirectory>webapps/caf-audit-service-admin</outputDirectory>
+                                                <includes>
+                                                    <include>com.github.cafaudit:caf-audit-service-admin</include>
```

Comment:
Shouldn't you also be adding this to the `excludes` block below like the others?

Reply:
done